data/common.yaml

```diff
@@ -0,0 +1,13 @@
+---
+profile_mysql_server::create_mysql_home: true
+profile_mysql_server::dbs: {}
+profile_mysql_server::mysql_gid: 238
+profile_mysql_server::mysql_groupname: "mysql"
+profile_mysql_server::mysql_home: "/var/lib/mysql"
+profile_mysql_server::mysql_logdir: "/var/log/mariadb"
+profile_mysql_server::mysql_parentdir: null
```

Comment:
I do not quite get what `mysql_parentdir` is supposed to be.

Is this the parent directory of `mysql_home`? Or something different?